### PR TITLE
Adding date picker Screen 

### PR DESCRIPTION
**Pull request by spiderxm**
Switch between dates. 
WhatsApp Image 2021-01-10 at 5 35 23 PM

### CODE REVIEW

File: lib/screens/activity_edit_sheet.dart [hunk 1]
```diff
@@ -1,20 +1,24 @@
 import 'package:flutter/cupertino.dart' as c;
+import 'package:flutter/material.dart';
```
Comment (hashirshoaeb) [inline lib/screens/activity_edit_sheet.dart:2]: not using material

File: lib/screens/activity_edit_sheet.dart [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
 import 'package:flutter/widgets.dart';
+import 'package:star_book/screens/date_picker_sheet.dart';
```
Comment (hashirshoaeb) [inline lib/screens/activity_edit_sheet.dart:4]: relative import

File: lib/screens/activity_edit_sheet.dart [hunk 3]
```diff
@@ -66,6 +70,22 @@ class _ActivityEditSheetState extends State<ActivityEditSheet> {
     setState(() {});
   }
 
+  void onDateTap() async {
+
+    dynamic date = await Navigator.of(context).push(MaterialPageRoute(
```
Comment (hashirshoaeb) [inline lib/screens/activity_edit_sheet.dart:75]: add the route in rout_generator.dart and use named route.

File: lib/screens/date_picker_sheet.dart [hunk 4]
```diff
@@ -0,0 +1,119 @@
+import 'package:flutter/cupertino.dart' as c;
+import 'package:flutter/material.dart';
```
Comment (hashirshoaeb) [inline lib/screens/date_picker_sheet.dart:2]: no need of material.

**Review by hashirshoaeb — CHANGES_REQUESTED** [review]

File: lib/screens/date_picker_sheet.dart (hunk 4 above)
Comment (spiderxm) [reply lib/screens/date_picker_sheet.dart]: Done

File: lib/screens/activity_edit_sheet.dart (hunk 3 above)
Comment (spiderxm) [reply lib/screens/activity_edit_sheet.dart]: Done

**Review by hashirshoaeb — APPROVED** [review]

**Pull request merged**